jest_config/setup.js

```diff
@@ -3,7 +3,7 @@ csrf.name = 'csrfmiddlewaretoken';
 csrf.value = 'csrfmiddlewaretoken';
 global.document.body.append(csrf);
 global.window.Urls = new Proxy({}, {
-  get() {
-    return () => undefined;
+  get(obj, prop) {
+    return () => prop;
```

Comment:
what's this all about?

Reply:
This is a global mock for the Urls object, I made it return the prop requested (the name of the URL) so that in debugging, you can more easily see by inspection which url has been requested.